tests/e2e/specs/admin/4-refund-label.test.js

```diff
@@ -27,36 +26,25 @@ describe( 'Refund shipping label', () => {
 
 	it( "works if the label hasn't been used", async () => {
 		await withShippingLabelAndOrder( async ( { order } ) => {
-
 			await StoreOwnerFlow.openExistingOrderPage( order.id );
 
 			await clickReactButton( '.ellipsis-menu > .button' );
 			await clickReactButton( 'div > .popover > .popover__inner > .popover__menu > .popover__menu-item:nth-child(3)' );
-			await clickReactButton( '.ReactModalPortal > .ReactModal__Overlay > .ReactModal__Content > .dialog__action-buttons > .button:nth-child(2)' );
```

Comment:
I'm puzzled by the old behavior - the `newLabelButton` is not present to create a shipping label; and the purchase flow does not present a "purchase" button, but it requires the whole address verification and package purchase.
So I'm deleting this, we're just verifying that the refund process works.